### ISSUE TITLE
Does it support bulk uploading or importing folders?

### ISSUE THREAD

**Issue by lxxself** [dzikoysk/reposilite#1756]
### Request details

The current upload seems to only allow uploading packages one by one. Have I missed a way to upload them in batches?
I hope batch uploading can be supported.
Also, I hope direct folder uploading can be supported.

**Comment by dzikoysk** [comment]
Unfortunately, it's not supported atm. There's always an option to upload it through ftp to your machine, we don't have any kind of cache, so it should be instantly picked up by Reposilite.

**Comment by lxxself** [comment]
> Unfortunately, it's not supported atm. There's always an option to upload it through ftp to your machine, we don't have any kind of cache, so it should be instantly picked up by Reposilite.

Yes, I tried copying the folder directly to the corresponding repository directory. It works, but the front-end page displays with some things missing, such as configuration information for maven, gradle, etc. on the right side

**Comment by dzikoysk** [comment]
The configuration data is extracted from corresponding maven-metadata.xml files, so if you didn't upload them then it won't work
